exercises/practice/rational-numbers/.meta/example.zig

```diff
@@ -0,0 +1,74 @@
+const std = @import("std");
+
+pub const RationalError = error{
+    InitializationFailure,
+};
+
+pub const Rational = struct {
+    numerator: i64,
+    denominator: i64,
+
+    pub fn init(numer: i64, denom: i64) RationalError!Rational {
+        if (denom == 0) return RationalError.InitializationFailure;
```

Comment:
Not sure if it's the right thing here to return an error as other language tracks do. I had to use many try catch blocks in tests.